apps/client/src/routes/authored/actions.ts

```diff
@@ -0,0 +1,31 @@
+import { parseFormData, validateFormData } from "bugs-lib/formData";
+import { z } from "zod";
+import { updateCourseChapter } from "$lib/data/courses";
+import { fail } from "@sveltejs/kit";
+
+const chapterSchema = z.object({
+  courseId: z.string(),
+  index: z.number(),
+  title: z.string(),
+  description: z.string(),
+});
+
+export const handleUpdateChapter = async ({ request }) => {
+  const parsedFormData = parseFormData(await request.formData());
+  const {
+    formData: { courseId, index, title, description },
```

Comment:
Don't forget to destructure `errors`. We don't have to display error messages for now.
But if you're interested in that, DaisyUI [input](https://daisyui.com/components/input/) lets you add error styles. You could also use [auto-animate](https://auto-animate.formkit.com/#examples) to improve how these errors are displayed.